packages/quantic/force-app/examples/main/lwc/exampleQuanticRefineModalContent/exampleQuanticRefineModalContent.html

```diff
@@ -0,0 +1,81 @@
+<template>
+  <c-example-layout
+    title={pageTitle}
+    description={pageDescription}
+    show-preview={isConfigured}
+  >
+    <c-configurator
+      slot="configuration"
+      options={options}
+      ontryitnow={handleTryItNow}
+    ></c-configurator>
+
+    <c-quantic-search-interface slot="preview" engine-id={engineId}>
+      <div class="facets-container">
+        <c-quantic-facet-manager engine-id={engineId}>
+          <c-quantic-facet
+            engine-id={engineId}
+            facet-id="objecttype"
+            field="objecttype"
+            label="Type"
+            number-of-values="6"
+            sort-criteria="alphanumeric"
+            no-search
+            display-values-as="link"
+            no-filter-facet-count
+            injection-depth="800"
+          >
+          </c-quantic-facet>
+          <c-quantic-numeric-facet
+            engine-id={engineId}
+            field="ytlikecount"
+            facet-id="ytlikecount"
+            label="Youtube Likes"
+            number-of-values="6"
+            sort-criteria="descending"
+            range-algorithm="even"
+            with-input="integer"
+            formatting-function={formattingFunction}
+          >
+          </c-quantic-numeric-facet>
+          <c-quantic-category-facet
+            engine-id={engineId}
+            facet-id="geographicalhierarchy"
+            field="geographicalhierarchy"
+            label="Country"
+            no-filter-by-base-path
+            no-filter-facet-count
+            delimiting-character=";"
+            number-of-values="6"
+            sort-criteria="alphanumeric"
+            with-search
+          >
+          </c-quantic-category-facet>
+          <c-quantic-timeframe-facet
+            engine-id={engineId}
+            facet-id="date"
+            field="date"
+            label="Date"
+            with-date-picker="true"
+            no-filter-facet-count
+            injection-depth="800"
+          >
+            <c-quantic-timeframe
+              period="next"
+              unit="year"
+            ></c-quantic-timeframe>
+            <c-quantic-timeframe amount="2" unit="week"></c-quantic-timeframe>
+            <c-quantic-timeframe unit="month"></c-quantic-timeframe>
+            <c-quantic-timeframe unit="year"></c-quantic-timeframe>
+            <c-quantic-timeframe
+              amount="10"
+              unit="year"
+              label="Last decade"
+            ></c-quantic-timeframe>
+          </c-quantic-timeframe-facet>
+        </c-quantic-facet-manager>
+      </div>
+      <c-quantic-refine-modal-content engine-id={engineId} hide-sort={config.hideSort}></c-quantic-refine-modal-content>
```

Comment:
for testing, I suggest to setup the page where `The Quantic-refine-modal is passed in the content slot of the Quantic-modal Component` _ like the way you show me, it's safer, as these 2 components should be used together (?)

Reply:
Actually the two components will be tested together in the tests of the [QuanticRefineToggle component](https://github.com/coveo/ui-kit/blob/055987fec3f4354a4143f7e8410514da89a65c40/packages/quantic/force-app/main/default/lwc/quanticRefineToggle/quanticRefineToggle.html#L17-L40). this component is the button that will be used to open the refine modal.
I will open[ the PR of the QuanticRefineToggle](https://github.com/coveo/ui-kit/pull/1970) after merging this one.